webpack/scenes/Content/Table/ContentTable.js

```diff
@@ -0,0 +1,118 @@
+import React, { useState, useCallback, useEffect, useMemo } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { STATUS } from 'foremanReact/constants';
+import { TableVariant, Thead, Tbody, Tr, Th, Td } from '@patternfly/react-table';
+import TableWrapper from '../../../components/Table/TableWrapper';
+import { getContent, getContentTypes } from '../ContentActions';
+import { selectContent, selectContentStatus, selectContentError, selectContentTypes, selectContentTypesStatus } from '../ContentSelectors';
+import SelectableDropdown from '../../../components/SelectableDropdown';
+import contentConfig from '../ContentConfig';
+import Loading from '../../../components/Loading';
+
+const ContentTable = () => {
+  const dispatch = useDispatch();
+  const status = useSelector(selectContentStatus);
+  const error = useSelector(selectContentError);
+  const response = useSelector(selectContent);
+  const enabledTypesResponse = useSelector(selectContentTypes);
+  const enabledTypesStatus = useSelector(selectContentTypesStatus);
+  const [contentTypeSelected, setContentTypeSelected] = useState('Python Packages');
```

Comment:
You could move this to the parent ContentPage and pass down as a prop..This will let the Header on Page change with type selection.

Reply:
cool!